content/sql/concepts/math-functions/math-functions.md

```diff
@@ -0,0 +1,19 @@
+---
+Title: 'Math Functions'
+Description: 'SQL Math Functions are built-in features in SQL, which allow mathematical operations directly in database queries.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'SQLite'
+  - 'MySQL'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+In SQL, **Math Functions** are built-in features which perform mathematical operations and complex calculations directly in database queries. This makes it possible to work with numerical data in SQL rather than extracting the data and processing it separately.
```

Comment:
```suggestion
In SQL, **math functions** are built-in features which perform mathematical operations and complex calculations directly in database queries. This makes it possible to work with numerical data in SQL rather than extracting the data and processing it separately.
```